tests/apt/test_protocol.py

```diff
@@ -215,6 +218,72 @@ def test_AptMessage_MGMSG_MOD_IDENTIFY_to_bytes() -> None:
     assert msg.to_bytes() == b"\x23\x02\x01\x00\x50\x01"
 
 
+def test_AptMessage_MGMSG_MOT_GET_POSCOUNTER_from_bytes() -> None:
+    msg = AptMessage_MGMSG_MOT_GET_POSCOUNTER.from_bytes(
+        bytes.fromhex("1204 0600 A2 01 0100400D0300")
+    )
+
+    assert msg.destination == 0x22
```

Comment:
```suggestion
    # The official documentation's example refers to this as "Channel 2," but this is actually bay 1.
    assert msg.destination == 0x22
```

Reply:
Help. It seems like if I try to commit this, the tests don't pass anymore. Should I add this comment locally, run `black`, and re-commit it (from my own pc)?

Reply:
In general, you should assume that I don't usually check any of my suggested code changes for correctness and they need to be tested yourself. In this case, I forgot the indentation.

`git pull` your branch locally to get the changes that were committed when you added this suggestion, reformat the code, commit and push it.

Reply:
Oh yes. I see. That makes sense. 

Alright. I have done the deeds. Thank you as always for reviewing my code and providing very insightful comments. I definitely have learned a lot so far!